src/model.jl

```diff
@@ -573,3 +353,148 @@ function read_geography(filepath="The_World.dat",nlongitude=128,nlatitude=65)
+const perihelion_parameters_mean_rate = [31.609974, 32.620504, 24.172203, 0.636717, 31.983787,
+                                         3.138886, 30.973257, 44.828336, 0.991874, 0.373813,
+                                         43.668246, 32.246691, 30.599444, 2.147012, 10.511172,
+                                         42.681324, 13.650058, 0.986922, 9.874455, 13.013341,
+                                         0.262904, 0.004952, 1.142024, 63.219948, 0.205021,
+                                         2.151964, 64.230478, 43.836462, 47.439436, 1.384343,
+                                         7.437771, 18.829299, 9.500642, 0.431696, 1.16009,
+                                         55.782177, 12.639528, 1.155138, 0.168216, 1.647247,
+                                         10.884985, 5.610937, 12.658184, 1.01053, 1.983748,
+                                         14.023871, 0.560178, 1.273434, 12.021467, 62.583231,
+                                         63.593761, 76.43831, 4.28091, 13.218362, 17.818769,
+                                         8.359495, 56.792707, 8.448301, 1.978796, 8.863925,
+                                         0.186365, 8.996212, 6.771027, 45.815258, 12.002811,
+                                         75.27822, 65.241008, 18.870667, 22.009553, 64.604291,
+                                         11.498094, 0.578834, 9.237738, 49.747842, 2.147012,
+                                         1.196895, 2.133898, 0.173168]
+const perihelion_parameters_phase = [251.9025, 280.8325, 128.3057, 348.1074, 292.7252, 165.1686,
+                                     263.7951, 15.3747, 58.5749, 40.8226, 308.4258, 240.0099,
+                                     222.9725, 106.5937, 114.5182, 268.7809, 279.6869, 39.6448,
+                                     126.4108, 291.5795, 307.2848, 18.93, 273.7596, 143.805,
+                                     191.8927, 125.5237, 172.7351, 316.7998, 319.6024, 69.7526,
+                                     123.5968, 217.6432, 85.5882, 156.2147, 66.9489, 20.2082,
+                                     250.7568, 48.0188, 8.3739, 17.0374, 155.3409, 94.1709, 221.112,
+                                     28.93, 117.1498, 320.5095, 262.3602, 336.2148, 233.0046,
+                                     155.6977, 184.6277, 267.2772, 78.9281, 123.4722, 188.7132,
+                                     180.1364, 49.1382, 152.5268, 98.2198, 97.4808, 221.5376,
+                                     168.2438, 161.1199, 55.0196, 262.6495, 200.3284, 201.6651,
+                                     294.6547, 99.8233, 213.5577, 154.1631, 232.7153, 138.3034,
+                                     204.6609, 106.5938, 250.4676, 332.3345, 27.3039]
+
+
+
+# Eqn. 4 from Berger 1978, Long-Term Variations of daily Insolation and Quaternary Climatic Changes
+function e_sincos_pi(year)
+  M = eccentricity_parameters_amplitude
+  g = eccentricity_parameters_mean_rate
+  β = eccentricity_parameters_phase
```

Comment:
Hm, do we really want to use unicode stuff in the code? 
So far, we have the same style guide as in Trixi: no unicode stuff in the code, but only in the comments or documentation
Uah, I just saw, that this is used throughout all of the new code, even epsilon+...ha ha ha, maybe that is a bit too much?!

Reply:
I agree, it's contra our usual conventions. However, my latest info was that we want to do things differently here and try to adhere to the notation used in the paper(s), but maybe that is out of date.

I only used the unicode symbols internally, to make it easier to directly recognize the relevant equations in the paper. However, I am also fine to change it back. Maybe I can create an issue for this and the final decision can be taken when the big cleanup day has come?

Reply:
I'm ok with or without the unicode code. You are the ones who know what the "good practices" in julia are :)

Reply:
I created an issue for it: #8.